viite-UI/src/model/RoadNameCollection.js

```diff
@@ -1,10 +1,13 @@
 (function (root) {
     root.RoadNameCollection = function (backend) {
```

Comment:
The collection will need to be changed to start having json object with fields instead of `editedField`